02 Sample App/01 Login/src/rest-api/login/loginAPI.ts

```diff
@@ -0,0 +1,19 @@
+import {LoginCredential} from '../../models/loginCredential';
+import {UserProfile} from '../../models/userProfile';
+import {userProfiles} from './loginMockData';
+
+class LoginAPI {
```

Comment:
Add a comment indicating that this is a fake api, but it returns promises could be replaced by a real one